projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -56,22 +58,43 @@ export default function SubscriptionControls( {
 } ) {
 	return (
 		<>
+			{ subscriberCount > 1 && (
+				<div
+					style={ {
+						margin: 20,
+						padding: 24,
+						background: '#f2f2f2',
+						borderRadius: '4px',
+						display: 'flex',
+						alignItems: 'center',
+						justifyContent: 'space-between',
+					} }
+				>
+					<span style={ { marginRight: 12 } }>
+						{ createInterpolateElement(
+							sprintf( '<span>%s readers</span> are\xA0subscribed', subscriberCount, 'jetpack' ),
```

Comment:
We'll want for this to be translatable, with a translator comment.

Reply:
Noting that even if you're excluding singular, some languages are more complex, and you'll probably still need to use `_n()`.

Reply:
Using`_n()` now. https://github.com/Automattic/jetpack/pull/22694/commits/4c794302f128f891b52731cacf0cdcf75bbc45f6